.dependabot/config.yml

```diff
@@ -17,7 +17,7 @@ update_configs:
           dependency_name: "jquery"
       - match:
           dependency_name: "@types/node"
-          version_requirement: "^10.12.18"
+          version_requirement: "^14.6.1"
```

Comment:
Should we just remove `@types/node` from the dependabot ignore list?

Reply:
IMO, we should, I think that is there because in the pass, that package gave us a lot of issue, but looks like it is working fine now.

Reply:
do you want me to remove that?

Reply:
Yes please, thanks!